.github/workflows/release.yml

```diff
@@ -0,0 +1,108 @@
+name: DiAGRAM release to live
```

Comment:
Should we add quotes around "live"?

Reply:
I don't think we need to, the meaning is fairly clear.